packages/design-system/src/components/web-components/ds-drawer/ds-drawer.stories.tsx

```diff
@@ -0,0 +1,195 @@
+import type { Meta } from '@storybook/react';
+import { useEffect, useState } from 'react';
+import { action } from '@storybook/addon-actions';
+import WebComponentDocTemplate from '../../../../../../.storybook/docs/WebComponentDocTemplate.mdx';
+import { webComponentDecorator } from '../storybook';
+import './ds-drawer';
+import '../ds-button';
+
+const meta: Meta = {
+  title: 'Web Components/ds-drawer',
+  decorators: [webComponentDecorator],
+  parameters: {
+    docs: {
+      page: WebComponentDocTemplate,
+      description: {
+        component:
+          'For information about how and when to use this component, [refer to its full documentation page](https://design.cms.gov/components/drawer/).',
+      },
+      slots: {
+        'footer-body': {
+          description:
+            'Slot for displaying custom content in the footer area of the Drawer. This can include text, or HTML elements. Use this slot to provide additional contextual information, actions, or links within the Drawer footer.',
+        },
+      },
+      componentEvents: {
+        'ds-close-click': {
+          description:
+            "A callback function triggered when the user clicks the close button or presses the ESC key, provided focus trapping is active. The visibility of the drawer is managed by the parent component, which must utilize this callback to toggle the drawer's display. The drawer itself does not automatically hide.",
+        },
+      },
+    },
+  },
+  argTypes: {
+    'aria-label': {
+      description: 'Gives more context to screen readers on the Drawer close button.',
+      control: 'text',
+    },
```

Comment:
Is this going to get the same treatment that @jack-ryan-nava-pbc's is getting where it gets a unique name so it doesn't get applied to both the root and child element in the a11y tree?

Reply:
@pwolfert I think it might. We were just discussing that [here](https://github.com/CMSgov/design-system/pull/3256#discussion_r1784886379)